examples/server/lwm2mserver.c

```diff
@@ -992,6 +1181,9 @@ command_desc_t commands[] = {{"list", "List registered clients.", NULL, prv_outp
 
                              COMMAND_END_LIST};
 
+/**
+ * @brief Starts the CoAP server.
+ */
 void startCoapServer() {
```

Comment:
we should add comments explaining what each conditional and imported function do in the process of `startCoapServer`. This should be documented in details especially since this is key procedure that we are going to expose to NodeJS server.